osu.Game/Screens/Ranking/Expanded/StarRatingDisplay.cs

```diff
@@ -24,27 +24,42 @@ public class StarRatingDisplay : CompositeDrawable
     {
         private readonly BeatmapInfo beatmap;
 
+        private StarDifficulty? difficulty;
```

Comment:
Not entirely sure I like both these fields existing, but I also don't see a good way to eliminate either, so...